vercel.json

```diff
@@ -1,6 +1,12 @@
 {
-  "crons": [{
-    "path": "/api/v1/cron.uploadRawTranscripts",
-    "schedule": "30 9 * * *"
-  }]
-}
+  "crons": [
+    {
+      "path": "/api/v1/cron.uploadRawTranscripts",
+      "schedule": "30 9 * * *"
+    },
+    {
+      "path":"/api/v1/cron.currentMeetingNumbers",
+      "schedule": "2 * * * *"
```

Comment:
fwiw, the free vercel version doesn't allow cron jobs more frequently than once a day. we can change it to a minute or two once we get vercel's open source discount.

Reply:
Actually you need to change this:


Deployment failed with the following error:

`Hobby accounts are limited to daily cron jobs. This cron expression (2 * * * *) would run more than once per day. Upgrade to pro to unlock all Cron Jobs features on Vercel.`